translator/translator_test.go

```diff
@@ -75,14 +74,14 @@ func TestFullyPopulatedModel(t *testing.T) {
 		},
 		TimestampKey: ptypes.TimestampNow(),
 	}
-	entity, err := ProtoMessageToDatastoreEntity(srcProto)
+	entity, err := ProtoMessageToDatastoreEntity(srcProto, true)
```

Comment:
I don't see any test case which verifies that underscore_separated field names are used on the generated Entity object when providing ``false`` for ``snakeCase`` argument.

Or am I missing something?

And ideally there would also be a test case for the whole round-trip.